nbs/Paper.qmd

```diff
@@ -0,0 +1,82 @@
+
+For our experiments, we used the "easy split" setup for the dataset, where 70% of the image patches were used for training, 20% for validation, and 10% for testing. This ensured reproducibility and allowed for a comprehensive comparison with existing learning-based hyperspectral image compression techniques.
+
+
+
+#### 2.2 Spectral Signals Compressor Network (SSCNet)
+
+SSCNet is a deep learning-based compression model that utilizes 2D convolutional layers with parametric ReLU activations for both spatial and spectral compression [4]. The encoder network employs 2D convolutions and three max pooling layers to achieve a fixed spatial compression factor of 64. The decoder mirrors the encoder, with upsampling achieved through transposed convolutional layers.
+
+The compression ratio (CR) is set by the number of latent channels in the bottleneck layer, while the network's ability to reconstruct the original image with minimal distortion is evaluated through Peak Signal-to-Noise Ratio (PSNR). SSCNet was chosen for this study due to its effectiveness in combining both spatial and spectral compression in hyperspectral datasets.
+
+#### 2.3 FasterAI Static Quantization
+
+To enhance the performance of SSCNet, we implemented static quantization from FasterAI, which is particularly well-suited for optimizing deep learning models for inference on resource-constrained hardware. Static quantization works by representing network weights and activations using lower precision (e.g., 8-bit integers), significantly reducing the memory footprint and computational requirements during the inference stage. This makes the model more efficient while maintaining acceptable performance levels in terms of reconstruction quality.
+
+In our work, we integrated FasterAI static quantization into SSCNet, applying it after the network was fully trained. This process required calibration using the training dataset to determine optimal quantization parameters that minimize information loss while allowing efficient compression.
+
+### 3. Experimental Setup
+
+The experimental setup used PyTorch as the deep learning framework. Gradient clipping and min-max normalization were applied to scale input data within a range suitable for learning-based models. The Adam optimizer was used with an initial learning rate of $10^{-5}$, which was decreased as training progressed to achieve convergence.
+
+Our experiments were conducted on a high-performance computing environment, featuring an NVIDIA A100 GPU with 80 GB memory. We trained the SSCNet model for 2000 epochs, followed by FasterAI static quantization. This setup aimed to balance training time and optimal model performance.
+
+
+
+![](./images/Pipeline.png)
```

Comment:
Great figure. 
It would be nice if you can make the pipeline clearer. For example, consider adding a new column for model compression. Ideally, the audience should be able to understand the figure without needing to read the paper in detail.